pkg/metrics/metrics.go

```diff
@@ -0,0 +1,119 @@
+// Copyright 2019 The etcd Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	// LabelSucceeded is a metric label indicating weather associated metric
+	// series is for success of failure.
+	LabelSucceeded = "succeeded"
+	// ValueSucceededTrue is value True for metric label succeeded.
+	ValueSucceededTrue = "true"
+	// ValueSucceededFalse is value False for metric label failed.
+	ValueSucceededFalse = "false"
+	// LabelKind is a metrics label indicates kind of snapshot associated with metric.
+	LabelKind = "kind"
+)
+
+var (
+	namespaceEtcdBR   = "etcdbr"
+	subsystemSnapshot = "snapshot"
+
+	// GCSnapshotCounter is metric to count the gabage collected snapshots.
+	GCSnapshotCounter = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: namespaceEtcdBR,
+			Subsystem: subsystemSnapshot,
+			Name:      "gc_total",
+			Help:      "Number garbage collected snapshots.",
```

Comment:
s/Number garbage collected snapshots/Total number of garbage collected snapshots/
For the sake of consistency with the documentation.